### PR TITLE
refactored templates by splitting it into partials

### PR DESCRIPTION
**Pull request by jordythevulder**
Made it easier to overwrite templates by splitting the frontend into multiple views

### CODE REVIEW

File: resources/views/cart/partials/products.blade.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <table class="w-full border-b">
-    <thead class="bg-ct-inactive-100 mt-5 flex rounded md:table-header-group">
+    <thead class="bg-ct-inactive-100 flex rounded md:table-header-group">
```
Comment (Jade-GG) [inline resources/views/cart/partials/products.blade.php:2]: This margin is no longer necessary? Not sure why this is part of this refactoring as the structure changes don't affect this specifically as far as I can tell.

File: resources/views/cart/coupon.blade.php [hunk 2]
```diff
@@ -2,20 +2,7 @@
...
-                        v-on="inputEvents"
-                        v-bind:value="couponCode"
-                        v-bind:disabled="$root.loading"
-                        required
-                    />
-                    <x-rapidez-ct::button.outline type="submit" loader>
-                        @lang('Apply')
-                    </x-rapidez-ct::button.outline>
-                </div>
+                @include('rapidez-ct::cart.partials.coupon-field')
```
Comment (indykoning) [inline resources/views/cart/coupon.blade.php:5]: I think this input variation could be utilised best as a component, so classes and other attributes could be passed if necessary

File: resources/views/cart/partials/products.blade.php (hunk 1 above)
Comment (jordythevulder) [reply resources/views/cart/partials/products.blade.php:2]: As i've mentioned in my other PR, the margin should be set by the Section component, and not individually on this element.

File: resources/views/cart/partials/coupon-field.blade.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<div class="flex w-full flex-row gap-x-3 checkout-theme">
```
Comment (Roene-JustBetter) [inline resources/views/cart/partials/coupon-field.blade.php:1]: Why do you need the class checkout-theme here?

**Comment by royduin** [comment]
@anyone

**Comment by jordythevulder** [comment]
These additions are compatible for the 1.x Rapidez version. Note to self: There will be a follow-up Pullrequest to make this compatible with the V2 version

**Review by indykoning — APPROVED** [review]

**Pull request merged**